python/pyspark/sql/functions.py

```diff
@@ -2548,6 +2549,10 @@ def pandas_udf(f=None, returnType=None, functionType=None):
        |  2|6.0|
        +---+---+
 
+       .. note:: If returning a new `pandas.DataFrame` constructed with a dictionary, it is
```

Comment:
Can we do a "warning"?

Reply:
I don't know if there is warning, but the notes show up highlighted so it's pretty clear

Reply:
K. I think sphinx supports these http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#directives

"caution" or "warning" seems more appropriate but note works too.